includes/namespace.php

```diff
@@ -100,18 +148,16 @@ function _print_missing_build_admin_notice() {
  * @return void
  */
 function activate( $network_wide = false ) {
-	if ( version_compare( PHP_VERSION, WEBSTORIES_MINIMUM_PHP_VERSION, '<' ) ) {
-		wp_die(
-		/* translators: %s: PHP version number */
-			esc_html( sprintf( __( 'Web Stories requires PHP %s or higher.', 'web-stories' ), WEBSTORIES_MINIMUM_PHP_VERSION ) ),
-			esc_html__( 'Plugin could not be activated', 'web-stories' )
-		);
-	}
+	global $web_stories_compatibility;
 
-	if ( version_compare( get_bloginfo( 'version' ), WEBSTORIES_MINIMUM_WP_VERSION, '<' ) ) {
+	$web_stories_compatibility->check_php_version();
```

Comment:
Since the `Compatibility` class is using a namespace, this check will fail if someone tries to use PHP 5.2. This is why the AMP plugin has avoided using a class thus far. Granted, now that WordPress core has PHP version checking built-in for plugins, this is less of a concern.

Reply:
There were already namespaces functions and classes in this file. This is not something we supported. The compability class does not need to be namespaced, I can prefix the name of the class.